ezgui/src/canvas.rs

```diff
@@ -142,6 +146,24 @@ impl Canvas {
         } else if self.drag_just_ended {
             self.drag_just_ended = false;
         }
+
+        let cursor_screen_pt = self.get_cursor().to_pt();
+        let cursor_map_pt = self.screen_to_map(self.get_cursor());
+        let inner_bounds = self.get_inner_bounds();
+        let map_bounds = self.get_map_bounds();
+        if !inner_bounds.contains(cursor_screen_pt) && !self.disable_edge_panning
```

Comment:
This behavior interacts strangely when dragging the map to pan. If I start dragging and happen to move my cursor to the edge of the screen, this panning also kicks in. I think all of this new behavior should be an `else` block after `else if self.drag_just_ended`